mysql-test/suite/parts/t/partition_icp.test

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is this file really copied from MySQL? It doesn't look so, please remove the copyright...

Reply:
Yes, it is copied from mysql-server sha 67d2e4e but I added some new tests to the end of it.

Reply:
I'd include this reference and credit to the author/company in the commit message.

Reply:
For what it is worth, the MySQL test file was last changed in mysql/mysql-server@f966d5d5617f5805a08942aac297a05693ef6128 and there is no copyright notice. From my time at Oracle, I remember that there was a policy that no copyright notices should be added to public MySQL test files.